prow/jobs/community/community-governance.yaml

```diff
@@ -0,0 +1,54 @@
+presubmits: # runs on PRs
+  kyma-project/community:
+    - name: community-governance
+      skip_report: true
+      decorate: true
+      max_concurrency: 10
+      branches:
+        - master
+      labels:
+        preset-dind-enabled: "true"
+        preset-build-pr: "true"
+      run_if_changed: "milv.config.yaml|.md$"
+      path_alias: github.com/kyma-project/community
+      extra_refs:
+        - org: kyma-project
+          repo: test-infra
+          base_ref: master
+          path_alias: github.com/kyma-project/test-infra
+      spec:
+        containers:
+          - image: eu.gcr.io/kyma-project/prow/test-infra/bootstrap:v20181121-f3ea5ce
+            securityContext:
+              privileged: true
+            command:
+              - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/governance.sh"
+            args:
+              - "--repository community"
```

Comment:
Instead of spaces use `=`. Change it also in other places.

Reply:
I think that without `=` is better 😄